python/pyspark/pandas/series.py

```diff
@@ -992,8 +993,10 @@ def map(self, arg: Union[Dict, Callable]) -> "Series":
 
         Parameters
         ----------
-        arg : function or dict
+        arg : function, dict or pd.Series
```

Comment:
If we accept `pd.Series`, we might also want to accept `ps.Series`, which could be another future work.

Reply:
Makes sense, a TODO is added for now.